content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,131 @@
+* The "live coding" feature, a.k.a. `mvn compile quarkus:dev` shortens your development iterations - check
+  [this 2 mins video](https://www.youtube.com/watch?v=4lXSf8DBQkQ)
+* Easy to test using JUnit 5 extensions
+* Hides the complexity of the native compilation using [GraalVM](https://www.graalvm.org/) or
+  [Mandrell](https://github.com/graalvm/mandrel)
+
+Check our https://camel.apache.org/camel-quarkus/latest/user-guide/first-steps.html[User guide]!
+
+## Camel components coverage
+
+Since the [early beginnings](https://github.com/quarkusio/quarkus/tree/0.11.0/extensions/camel) the number of supported
+Camel bits went up substantially. As of Quarkus 1.0.0 and
+Camel 3.4, Camel Quarkus supports:
+
+* 155 out of 350 Camel components (44%)
+* 27 out of 43 Camel data formats (63%)
+* 13 out of 17 Camel languages (76%)
+* 17 out of 42 miscellaneous Camel components (40%)
+
+Some of these are supported in JVM mode only - please refer to the
+[complete list](https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html) for details.
+
+## Bootstrapping methods
+
+There are two ways to bootstrap and configure Camel Quarkus:
+
+1. Bare
+2. With Camel Main
+
+Both methods configure and start a Camel Context for you. The main difference is in how much the resulting Camel Context
+obeys the conventions known from Camel on other platforms. If you like to configure your routes using
+`camel.component.*` properties, then you need to add the `camel-quarkus-main` dependency and the properties
+will work as usual on Camel standalone or Camel SpringBoot. Of course, this brings some overhead. You may prefer this
+option if you used Camel before or if you are migrating from other platform.
+
+If you rather come from the Quarkus side and you never wrote Camel integrations before, you may choose to use the bare
+bootstrap and configure your routes using CDI.
+
+See the [Bootstrap section](https://camel.apache.org/camel-quarkus/latest/user-guide/bootstrap.html) of the User guide
+for more details.
+
+## CDI
+
+CDI plays a central role in Quarkus and Camel Quarkus offers a first class support for it too. You may use `@Inject`,
+`@ConfigProperty` and other annotations e.g. to inject beans and configuration values to your Camel RouteBuilder's.
+See the [CDI section](https://camel.apache.org/camel-quarkus/latest/user-guide/cdi.html) of the User guide for more
+details.
+
+## Native mode
+
+As mentioned above, Quarkus brings the option to compile an application to native executable. Compared to running on
+JVM, native applications start faster, require less memory at runtime and are also smaller on disk.
+
+However, the native compilation with GraalVM is all but easy and straightforward. First, there are
+[limitations](https://github.com/oracle/graal/blob/master/substratevm/LIMITATIONS.md) for what your code and all your
+dependencies may or may not do. Second, the native compiler needs to be configured through many options that may
```

Comment:
may or may not be easy -> not easy

I dont think anything with graalvm is easy for the average user.

Reply:
Yeah, shorter is better.